app/controllers/targets_controller.rb

```diff
@@ -15,8 +16,16 @@ def index
     render json: current_user.targets
   end
 
+  def destroy
+    render json: @target.destroy
+  end
+
   private
 
+  def set_target
```

Comment:
Try to avoid naming methods with `set, get` prefixes, you can [take a look at here for more info](https://github.com/rubocop-hq/ruby-style-guide/issues/423) and [ruby style guide](https://github.com/rubocop-hq/ruby-style-guide#classes--modules)